services/frontend/src/routes/block-producers/general-information-profile.js

```diff
@@ -202,7 +267,7 @@ const GeneralInformation = ({ classes, producer }) => {
             variant='subtitle1'
             className={classNames(classes.value, classes.subTitle)}
           >
-            - -
+            {_get(producer, 'average') || 0}
```

Comment:
![image](https://user-images.githubusercontent.com/5632966/75211370-4af4ae80-5749-11ea-81a5-a7b0556e622c.png)

@tetogomez  this average should be two decimal places